packages/calling/calling/src/Contacts/constants.ts

```diff
@@ -1,4 +1,20 @@
 export const CONTACTS_FILE = 'Contacts';
 export const CONTACT_FILTER = 'contacts';
+export const GROUP_FILTER = 'groups';
 export const ENCRYPT_FILTER = 'encrypt';
 export const USERS = 'Users';
+export const DEFAULT_GROUP_NAME = 'Other contacts';
+export const CONTACTS_SCHEMA = 'urn:cisco:codev:identity:contact:core:1.0';
+
+export const encryptedFields = [
+  'addressInfo',
+  // 'avatarURL',
+  'companyName',
+  'displayName',
+  'emails',
+  'firstName',
+  'lastName',
+  'phoneNumbers',
+  'sipAddresses',
+  'title',
+];
```

Comment:
you could make this as an enum and use this as key-value for what @arun3528 is suggesting. You can maybe convert it into array wherever needed as an array of encryptedFields.